clang/test/Driver/sycl-offload-static-lib-2.cpp

```diff
@@ -94,8 +106,11 @@
 
 /// test behaviors of static lib with no source/object
 // RUN: %clangxx -target x86_64-unknown-linux-gnu -fsycl -L/dummy/dir %t.a -### 2>&1 \
-// RUN:   | FileCheck %s -check-prefixes=STATIC_LIB_NOSRC
-// STATIC_LIB_NOSRC: clang-offload-bundler{{.*}} "-type=ao" "-targets=host-x86_64-unknown-linux-gnu" "-inputs=[[INPUTLIB:.+\.a]]" "-check-section"
+// RUN:   | FileCheck %s -check-prefixes=STATIC_LIB_NOSRC,STATIC_LIB_NOSRC_A
+// RUN: %clangxx -target x86_64-unknown-linux-gnu -fsycl -L/dummy/dir %t.lo -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefixes=STATIC_LIB_NOSRC,STATIC_LIB_NOSRC_LO
+// STATIC_LIB_NOSRC_A: clang-offload-bundler{{.*}} "-type=ao" "-targets=host-x86_64-unknown-linux-gnu" "-inputs=[[INPUTLIB:.+\.a]]" "-check-section"
+// STATIC_LIB_NOSRC_LO: clang-offload-bundler{{.*}} "-type=ao" "-targets=host-x86_64-unknown-linux-gnu" "-inputs=[[INPUTLIB:.+\.lo]]" "-check-section"
```

Comment:
Just as a nicety:
```suggestion
// RUN:   | FileCheck %s -check-prefix=STATIC_LIB_NOSRC -DARCHIVE_EXT=a
// RUN: %clangxx -target x86_64-unknown-linux-gnu -fsycl -L/dummy/dir %t.lo -### 2>&1 \
// RUN:   | FileCheck %s -check-prefix=STATIC_LIB_NOSRC -DARCHIVE_EXT=lo
// STATIC_LIB_NOSRC: clang-offload-bundler{{.*}} "-type=ao" "-targets=host-x86_64-unknown-linux-gnu" "-inputs=[[INPUTLIB:.+\.]][[ARCHIVE_EXT]]" "-check-section"
```

Reply:
I like it - made changes, not exact to what you have above, but close.

Reply:
Better yet!